### DISCUSSION TITLE
I found an error in Python Quiz !!!

### DISCUSSION THREAD

**Discussion by 12Jesse24** [1101-GS4719-C/Python-Community-GS4719-C#1578] [Q&A] [answered]
The question is following

x = 3.14
x = x * 5
x = x // 2

what value is x?
The answer shows 7
Because the type of x is float,  I deem that this answer must be 7.0 not 7

https://ncuedu.tw/learning/chatgpt/cid/1/qid/32

**Comment by lazywulf** [comment, marked as answer]
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/103130674/b0c4ca7a-a522-4bbf-bf02-d940ceeadcb6)
Can confirm.

**Reply by chia-kai-chang** [reply]
謝謝 @12Jesse24 @lazywulf ，答案已修正了。
https://ncuedu.tw/learning/chatgpt/cid/1/qid/32

**Comment by yuchien-lin** [comment]
I thought you are right, i got the same answer  
![9-27-1](https://github.com/NCU-GS4719-Python/Python-Community/assets/71421116/2d598185-74c8-4072-845c-355152195336)

**Comment by 1chooo** [comment]
```py
x = 3.14
x = x * 5
x = x // 2
```

```py
print(f"type of x: {type(x).__name__}; value of x: {x}")

# type of x: float; value of x: 7.0
```

All the answers above are correct. In the future, when posting code on GitHub, you can also try using code snippets, so you can easily copy and paste them.

By TA
